### DISCUSSION TITLE
Dark theme doesn't work with typography.

### DISCUSSION THREAD

**Discussion by Liumingxun** [saadeghi/daisyui#2927] [Q&A] [answered]
Reproduction repo [here](https://stackblitz.com/edit/daisyui-astro-pfmo9h). 

I used `theme controller` swap component to change theme, build-in others can be work but the prose block haven't change to dark. The expected effect should similar to [here](https://daisyui.com/docs/layout-and-typography/). I try same config with pure html still that situation.

Is some configuration wrong?

**Comment by saadeghi** [comment, marked as answer]
When you enable a `ds-` prefix for daisyUI, all class names which are coming from daisyUI get a prefix. including the additional style we add for `prose`.  
So in this case you need to add an `ds-prose` to `prose` so the required colors get added to `prose`.

Let me know if you have a question.

**Comment by Liumingxun** [comment]
additionally, like this config can just only using `ds-prose` without `prose`
```ts
export default {
  daisyui: {
    prefix: 'ds-'
  },
  plugins: [
    typography({
	    className: 'ds-prose'
    }),
    daisyui
  ],
}

```
